.ai/pages/node-infrastructure-run-a-collator-collator.md

```diff
@@ -0,0 +1,547 @@
+
+## Obtain Chain Specification
+
+Download the chain specification for your target system parachain using one of the following options:
+
+### Download from Chainspec Collection (Recommended)
+
+Follow these steps to download your specification from the Chainspec Collection:
+
+1. Visit the [Chainspec Collection](https://paritytech.github.io/chainspecs/){target=\_blank} website.
+2. Find your target system parachain.
+3. Download the chain specification JSON file.
+4. Save it as `chain-spec.json`.
+
+### Build Chainspec from Runtime
+
+Follow these steps to build a chainspec from the runtime:
+
+1. Clone the runtimes repository and navigate into it using the following commands:
+  ```bash
+  git clone https://github.com/polkadot-fellows/runtimes.git
+  cd runtimes
+  ```
+
+2. Build the desired runtime. Use the following command for Polkadot Hub:
+  ```bash
+  cargo build --release -p asset-hub-polkadot-runtime
+  ```
+
+3. Install the `chain-spec-builder` dependency using the following command:
+  ```bash
+  cargo install --locked staging-chain-spec-builder@14.0.0
+  ```
+
+4. Finally, generate the chain spec using the following commands:
+  ```bash
+  chain-spec-builder create \
+    --relay-chain polkadot \
+    --para-id 1000 \
+    --runtime target/release/wbuild/asset-hub-polkadot-runtime/asset_hub_polkadot_runtime.compact.compressed.wasm \
+    named-preset production > chain-spec.json
+  ```
+
+??? tip "System Parachain Para IDs"
+
+    - Polkadot Hub: 1000
+    - Bridge Hub: 1002
+    - People Chain: 1004
+    - Coretime Chain: 1005
+
+## Create User and Directory Structure
+
+1. Create a dedicated user with the following command:
+  ```bash
+  sudo useradd -r -s /bin/bash polkadot
+  ```
+
+2. Use the following command to copy your chain spec to the directory:
+  ```bash
+  sudo cp chain-spec.json /var/lib/polkadot-collator/
+  ```
+
+3. Set permissions using the following command:
+  ```bash
+  sudo chown -R polkadot:polkadot /var/lib/polkadot-collator
+  ```
+
+## Create Systemd Service File
+
+1. Create a service file to hold the configuration for your collator:
+  ```bash
+  sudo nano /etc/systemd/system/polkadot-collator.service
+  ```
+
+2. Open the new file and add the following configuration code:
+  ```ini title="systemd/system/polkadot-collator.service"
+  [Unit]
+  Description=Polkadot System Parachain Collator
+  After=network.target
+
+  [Service]
+  Type=simple
+  User=polkadot
+  Group=polkadot
+  WorkingDirectory=/var/lib/polkadot-collator
+
+  # Block-Producing Collator Configuration
+  ExecStart=/usr/local/bin/polkadot-parachain \
+    --collator \
+    --chain=/var/lib/polkadot-collator/chain-spec.json \
+    --base-path=/var/lib/polkadot-collator \
+    --port=30333 \
+    --rpc-port=9944 \
+    --prometheus-port=9615 \
+    --node-key-file=/var/lib/polkadot-collator/node.key \
+    --name="YourCollatorName" \
+    -- \
+    --execution=wasm \
+    --chain=polkadot \
+    --port=30334 \
+    --sync=warp
```

Comment:
We don't need archive when running a collator - although I don't think warp sync works with pruned databases.  So i would add `--blocks-pruning=256`,  `--state-pruning=256`, and replace warp with `--sync=fast` to the both the para and relay sides.  Also recommended if it's not already default `--database=paritydb` for both para and relay chains. 

Optionally for the **relay chain only**, perhaps `--pool-limit=0` although that may confuse them and we certainly don't want them to enable that on the parachain side.